src/core/thread/osthread.d

```diff
@@ -299,7 +299,8 @@ class Thread : ThreadBase
         }
         else version (Posix)
         {
-            pthread_detach( m_addr );
+            if (m_addr != m_addr.init)
```

Comment:
Do we need to apply the same fix for `version (Windows)` above?

Reply:
I don't have a windows machine to test with but if CloseHandle crashes with `m_hndl.init` as it's argument, we probably should.

Reply:
If test is correct, then you should 'seen failures on the Windows CI pipelines.

Reply:
It is fine then, AFAICS it should be triggering a `CloseHandle` call with `m_hndl.init`